tests/syscall.rs

```diff
@@ -31,10 +31,18 @@ fn syscall() -> CrabResult<()> {
         )
         .unwrap();
 
+    assert!(target.memory_maps()?.contains(&addr));
+
     for line in std::fs::read_to_string(format!("/proc/{}/maps", target.pid()))?.lines() {
         if line.starts_with(&format!("{:08x}-", addr)) {
```

Comment:
By the way it looks like this test was created before `memory_maps()` was introduced. With the assert you introduced this for and if is no longer necessary.

Reply:
yes, and for the second part, maybe